packages/gestalt-design-tokens/tokens/color/component.json

```diff
@@ -0,0 +1,116 @@
+{
+  "button": {
+    "color": {
+      "primary": {
+        "default": {
+          "value": "{color.background.primary.base.value}",
+          "comment": "Primary button background color"
+        },
+        "hover": {
+          "value": "{color.background.primary.strong.value}",
+          "comment": "Primary button background color on hover"
+        },
+        "active": {
+          "value": "#a3081a",
```

Comment:
I'm curious about these hex values. Should these use tokens instead?

Reply:
There's no way to reconcicle this hex colors to existing tokens and we are in the midst of reworking all the tokens...

I found the easiest to just have them be "single" and prevent adding new aliases... and make it harder to deprecate later on.